### DISCUSSION TITLE
macOS runner resources inconsistent in docs

### DISCUSSION THREAD

**Discussion by der-eismann** [community/community#26656] [Actions]
<p>Hey everyone, while researching for Jenkins alternatives I was scrolling to the GitHub Actions docs and found the resources for macOS machines to be inconsistent.<br>
On <a href="https://github.com/features/actions#pricing-details" rel="noopener nofollow ugc">this page</a> it says that macOS runners have 2 cores and 7 GB RAM.<br>
But <a href="https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources" rel="noopener nofollow ugc">in the docs</a> it says that macOS runners have 3 cores and 14 GB RAM.</p>
<p>Since this is quite a significant difference it would be great if someone could tell me what’s true.</p>
<p>Thanks!</p>

**Comment by Simran-B** [comment, marked as answer]
<p>I would say that the docs are correct, because they were updated to describe the macOS specs separately:</p>
<aside class="onebox githubcommit" data-onebox-src="https://github.com/github/docs/commit/6e072b619ba510c309cbc7043455312bf5130de9">
  <header class="source">

      <a href="https://github.com/github/docs/commit/6e072b619ba510c309cbc7043455312bf5130de9" target="_blank" rel="noopener nofollow ugc">github.com/github/docs</a>
  </header>

  <article class="onebox-body">
    <div class="github-row">
  <div class="github-icon-container" title="Commit">
    <svg width="60" height="60" class="github-icon" viewbox="0 0 14 16" aria-hidden="true"><path d="M10.86 7c-.45-1.72-2-3-3.86-3-1.86 0-3.41 1.28-3.86 3H0v2h3.14c.45 1.72 2 3 3.86 3 1.86 0 3.41-1.28 3.86-3H14V7h-3.14zM7 10.2c-1.22 0-2.2-.98-2.2-2.2 0-1.22.98-2.2 2.2-2.2 1.22 0 2.2.98 2.2 2.2 0 1.22-.98 2.2-2.2 2.2z"></path></svg>
  </div>

  <div class="github-info-container">
    <h4>
      <a href="https://github.com/github/docs/commit/6e072b619ba510c309cbc7043455312bf5130de9" target="_blank" rel="noopener nofollow ugc">Update for GH Actions hosted runners specification and usage (#17700)</a>
    </h4>

    <div class="github-info">
      <div class="date">
        committed <span class="discourse-local-date" data-format="ll" data-date="2021-02-08" data-time="20:59:50" data-timezone="UTC">08:59PM - 08 Feb 21 UTC</span>
      </div>

      <div class="user">
        <a href="https://github.com/AlenaSviridenko" target="_blank" rel="noopener nofollow ugc">
          <img alt="AlenaSviridenko" src="https://user-images.githubusercontent.com/7819991/181099087-c914f135-3f8a-47c8-9107-604855819a26.jpeg" class="onebox-avatar-inline" width="20" height="20">
          AlenaSviridenko
        </a>
      </div>

      <div class="lines" title="changed 2 files with 14 additions and 3 deletions">
        <a href="https://github.com/github/docs/commit/6e072b619ba510c309cbc7043455312bf5130de9" target="_blank" rel="noopener nofollow ugc">
          <span class="added">+14</span>
          <span class="removed">-3</span>
        </a>
      </div>
    </div>
  </div>
</div>

  <div class="github-row">
    <p class="github-body-container">* updated runner specification and usage guidelines

* removed extra line

*<span class="show-more-container"><a href="https://github.com/github/docs/commit/6e072b619ba510c309cbc7043455312bf5130de9" target="_blank" rel="noopener nofollow ugc" class="show-more">…</a></span><span class="excerpt hidden"> added screenshot

* Updated wording

Co-authored-by: Sarah Edwards &lt;skedwards88@github.com&gt;

Co-authored-by: Sarah Edwards &lt;skedwards88@github.com&gt;</span></p>
  </div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>But if in doubt, you can always create a test workflow and run <code>sysctl hw</code> and/or <code>system_profiler SPHardwareDataType</code>. They confirm that it’s 3 cores and 14 GB memory:</p>
<pre><code class="lang-auto">hw.ncpu: 3
hw.byteorder: 1234
hw.memsize: 15032385536
...
hw.logicalcpu: 3
hw.logicalcpu_max: 3
hw.packages: 1
hw.pagesize: 4096
hw.pagesize32: 4096
hw.physicalcpu: 3
hw.physicalcpu_max: 3
</code></pre>
<pre><code>  Model Name: Mac
  Model Identifier: VMware7,1
  Processor Name: Unknown
  Processor Speed: 3.33 GHz
  Number of Processors: 1
  Total Number of Cores: 3
  L2 Cache (per Core): 256 KB
  L3 Cache: 12 MB
  Memory: 14 GB
  ...</code></pre>

**Comment by jsoref** [comment]
<p>Yeah, the docs are definitely correct.</p>
<p>I have no idea how to complain about /features/, my default is to file an item to:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181099088-3a0444b9-992e-4a06-be34-4fe158984f78.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/" target="_blank" rel="noopener">GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>One of their search results for sales is:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/enterprise/contact">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/enterprise/contact" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/361;"><img src="https://user-images.githubusercontent.com/2119212/181099089-558728a3-e428-495d-bb75-9fb741decca1.png" class="thumbnail" width="690" height="361" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d8ab085d1a0e558974b1679f7831f40f39155e0_2_690x361.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d8ab085d1a0e558974b1679f7831f40f39155e0_2_1035x541.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d8ab085d1a0e558974b1679f7831f40f39155e0_2_1380x722.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/d/1d8ab085d1a0e558974b1679f7831f40f39155e0_2_10x10.png"></div>

<h3><a href="https://github.com/enterprise/contact" target="_blank" rel="noopener">GitHub Enterprise: a smarter way to work together</a></h3>

  <p>Whether you’re working solo or leading an enterprise, GitHub has everything you need to build and scale your team’s workflow. Choose the plan that’s right for your hosting environment and security ...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Arguably that’s a totally reasonable entrypoint, you’re trying to talk to a sales group and they’re presenting themselves for enterprises. They should want their information to be correct and appealing to enterprises, so let the enterprise sales team fix their content.</p>

**Comment by Simran-B** [comment]
<p><a class="mention" href="/liligalante">@liligalante</a> informed the marketing team :heart:</p>
